build.go

```diff
@@ -416,6 +420,7 @@ func ldflags() string {
 	b.WriteString(fmt.Sprintf(" -X main.version=%s", version))
 	b.WriteString(fmt.Sprintf(" -X main.commit=%s", getGitSha()))
 	b.WriteString(fmt.Sprintf(" -X main.buildstamp=%d", buildStamp()))
+	b.WriteString(fmt.Sprintf(" -X main.chromiumRevision=%s", chromiumRevision))
```

Comment:
why is this a build flag?

Reply:
because we need to know the version of the bundled chromium in order to find the correct path